.eslines.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Oh, it had a typo, sorry. This was fixed (and updated with new config) within the latest changes.